deploy.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if ! command -v jq >/dev/null; then
+  echo "Must have JQ installed"
+  exit 1
+fi
+
+if [ -z ${AUTH_USERNAME+set} ]; then
+  echo "Must set AUTH_USERNAME environment variable"
+  exit 1
+fi
+
+if [ -z ${AUTH_PASSWORD+set} ]; then
+  echo "Must set AUTH_PASSWORD environment variable"
+  exit 1
+fi
+
+if [ -z ${ROUTE_SERVICE_APP_NAME+set} ]; then
+  echo "Must set ROUTE_SERVICE_APP_NAME environment variable"
+  exit 1
+fi
+
+if [ -z ${ROUTE_SERVICE_NAME+set} ]; then
+  echo "Must set ROUTE_SERVICE_NAME environment variable"
+  exit 1
+fi
+
+if [ -z ${PROTECTED_APP_NAME+set} ]; then
+  echo "Must set PROTECTED_APP_NAME environment variable"
+  exit 1
+fi
+
+API_ENDPOINT="$(cf target | head -n1 | awk '{ print $3 }')"
```

Comment:
API_ENDPOINT is unused and I had to remove this line - did you want to evaluate it?

Reply:
No, I think it's leftover from another approach I took to working out the apps domain. Will remove.